bika/lims/browser/analysisrequest/publish.py

```diff
@@ -726,7 +729,7 @@ def __call__(self, ar, overwrite=False):
         verified = wasTransitionPerformed(ar, 'verify')
         if not overwrite and verified:
             # Prevent any error related with digest
-            data = ar.getDigest() if hasattr(ar, 'getDigest') else {}
+            data = {} #ar.getDigest() if hasattr(ar, 'getDigest') else {}
```

Comment:
Although I also want to remove the Digester, we should do that in a separate PR;)

Reply:
If I uncomment this my code won't work. Please advise.

Reply:
@xispa: The digester seems to cause too many problems, I would suggest that we remove that code for the core as well. IMO the digester is also responsible for having Analyses w/o acquisition chain and I could imagine that the issue regarding the missing site is also because of this. Caching persistent objects is not a good idea.

https://github.com/senaite/senaite.core/blob/master/bika/lims/browser/analysisrequest/publish.py#L915

Reply:
Agree the digester is causing too many problems, and I am ok to eventually remove it. But until we don't have a new-better machinery for reporting, we need to find a workaround to boost the generation of reports as a replacement of the digester. Some of our clients select ~50 or more ARs for report generation at once, so if we remove the digester, the instance gets freezed (or generation takes too long), which imo, is worst. So, if you want to leave this comment, leave it, but don't remove anything from the digester so meanwhile we can at least keep this single line uncommented for our clients. Another option is to add a key-value in the registry and check it's value here.

Reply:
Uncommented and it now works. Not idea why. I'll continue to watch on the test server